documentation/operators/src/.faq/mixnodes-faq.md

```diff
@@ -0,0 +1 @@
+# Mix Nodes
```

Comment:
There was a mistake in SUMMARY.md where the path to /faq was named as .faq instead of ./faq. This bug created the redundant dir with the files in it. I simply removed it.